c_glib/arrow-glib/reader.h

```diff
@@ -283,4 +283,56 @@ GArrowTable *garrow_csv_reader_read(GArrowCSVReader *reader,
                                     GError **error);
 
 
+/**
+ * GArrowJSONReadUnexpectedFieldBehavior:
+ * @GARROW_JSON_READ_IGNORE: Ignore other fields.
+ * @GARROW_JSON_READ_ERROR: Return error.
+ * @GARROW_JSON_READ_INFERTYPE: Infer a type.
+ *
+ * They are corresponding to `arrow::json::UnexpectedFieldBehavior` values.
+ */
+typedef enum {
+  GARROW_JSON_READ_IGNORE,
+  GARROW_JSON_READ_ERROR,
+  GARROW_JSON_READ_INFERTYPE,
```

Comment:
Could you add `_` between `INFER` and `TYPE`?

Reply:
I've fixed it.